### PR TITLE
CLDR-14016 Revise localeId canonicalization specification

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14016
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by macchiati** [comment]
There are a few small changes from the design doc, that I needed (noticed problems) when integrating. Although Frank would be the primary, I think he doesn't have authorization to approve CLDR tickets. So Peter, you should look it over for consistency with CLDR and Frank can review for formal correctness.

File: docs/ldml/tr35.html [hunk 1]
```diff
@@ -8564,6 +8564,268 @@ <h2><a name="Links_to_Other_Parts" href="#Links_to_Other_Parts"
...
+		    </li>
+      
+		    <li>else if source.field = {} and replacement.field ≠ {}
+		      <ul>
+		        <li>source.field = replacement.field</li>
+	          </ul>
+		    </li>
+      </ul>
+		  <p><em>Example:</em></p>
+		  <blockquote><p>source=ja-Latn- fonipa-hepburn-heploc</p>
```
Comment (pedberg-icu) [inline docs/ldml/tr35.html:8678]: Should probably fix the space after ja-Latn-, that could be confusing

File: docs/ldml/tr35.html [hunk 2]
```diff
@@ -8564,6 +8564,268 @@ <h2><a name="Links_to_Other_Parts" href="#Links_to_Other_Parts"
...
+		  <p>rule =&rdquo;&lt;languageAlias type="und-hepburn-heploc"</p>
+		  <p>replacement="und-alalc97"&gt;&rdquo;</p>
+		  <p>&nbsp;</p>
+		  <p>result=&rdquo;ja-Latn-alalc97-fonipa&rdquo; // note that CLDR canonical order of variants is alphabetical</p></blockquote>
+	  <h5 >Territory Exception</h5>
+		  <p>If the field = territory, and the replacement.field has more than one value, then look up the most likely territory* for the base language code (and script, if there is one). If that likely territory is in the list of replacements, use it. Otherwise, use the first territory in the list.</p>
+		  <p><em>Example:</em></p>
+	  <blockquote><p>source=ja-Latn- fonipa-hepburn-heploc</p>
```
Comment (pedberg-icu) [inline docs/ldml/tr35.html:8686]: Should probably fix the space after ja-Latn-

File: docs/ldml/tr35.html [hunk 3]
```diff
@@ -8564,6 +8564,268 @@ <h2><a name="Links_to_Other_Parts" href="#Links_to_Other_Parts"
...
+		  <p>replacement="und-alalc97"&gt;&rdquo;</p>
+	    <p>&nbsp;</p>
+	    <p>result=&rdquo;ja-Latn-alalc97-fonipa&rdquo; <em>// note that CLDR canonical order of variants is alphabetical</em></p>
+		  </blockquote>
+	  <h4>5. Canonicalizing Syntax</h4>
+		<p>To canonicalize the syntax of <em>source</em>: </p>
+		<ul>
+		  <li>Initial Script Subtag
+		    <ul>
+		      <li>If the first subtag has 4 letters, prepend the source with &quot;-und&quot;</li>
```
Comment (pedberg-icu) [inline docs/ldml/tr35.html:8697]: Shouldn't the prepend be "und-" ?

**Review by pedberg-icu — COMMENTED** [review]
Had a few minor comments...

File: docs/ldml/tr35.html (hunk 1 above)
Comment (macchiati) [reply docs/ldml/tr35.html:8678]: fixing 2 instances

File: docs/ldml/tr35.html (hunk 3 above)
Comment (macchiati) [reply docs/ldml/tr35.html:8697]: got it.

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**